### PR TITLE
redirect uninitialized device to onboarding automatically

### PR DESCRIPTION
**Pull request by mroz22**
Need to discuss this with UX first, dont merge.

### CODE REVIEW

File: packages/suite/src/actions/suite/suiteActions.ts [hunk 1]
```diff
@@ -79,8 +80,12 @@ export const selectFirstAvailableDevice = () => () => {};
 // export const handleDeviceEvent = (device: Device) => (dispatch: Dispatch, getState: GetState) => {
 export const handleDeviceConnect = (device: Device) => (dispatch: Dispatch, getState: GetState) => {
     const selected = getState().suite.device;
+    const { app } = getState().router;
     if (!selected) {
         dispatch(selectDevice(device));
+        if (device.mode === 'initialize' && app !== 'onboarding') {
+            goto('/onboarding');
```
Comment (slowbackspace) [inline packages/suite/src/actions/suite/suiteActions.ts]: please use `getRoute` from utils/suite/router.ts

Comment (mroz22) [reply packages/suite/src/actions/suite/suiteActions.ts]: Ok, fixed. Check it now.

File: packages/suite/src/utils/suite/router.ts [hunk 2]
```diff
@@ -1,12 +1,13 @@
 import routesConstants from '@suite-constants/routes';
+import { makePublicRouterInstance } from 'next/router';
```
Comment (slowbackspace) [inline packages/suite/src/utils/suite/router.ts]: it looks like you don' actually use it anywhere

**Review by slowbackspace — CHANGES_REQUESTED** [review]

File: packages/suite/src/utils/suite/router.ts [hunk 3]
```diff
@@ -89,7 +92,7 @@ export const getParams = (url: string) => {
     return params;
 };
 
-export const getRoute = (name: string, params?: { [key: string]: string }) => {
+export const getRoute = (name: AnyRouteName, params?: { [key: string]: string }) => {
```
Comment (slowbackspace) [inline packages/suite/src/utils/suite/router.ts]: nice!

File: packages/suite/src/utils/suite/router.ts (hunk 2 above)
Comment (mroz22) [reply packages/suite/src/utils/suite/router.ts]: Oh crap, fixed. Cant get autoimports from vs code work

**Pull request merged**